### PR TITLE
Add is_active field in UserFilter

### LINKED ISSUE #3183: Inactive users in users list
### My actions before raising this issue
- [X ] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [X ] Searched [past issues](/issues)

### Current Behaviour

There is users list limit, but that users list includes inactive users

### Possible Solution

Add filter to users list:
queryset = User.objects.prefetch_related('groups').all().filter(is_active=True).order_by('id')

### Your Environment

- cvat version - release v1.3.0

### PR DESCRIPTION
**Pull request by Jooong**
### Motivation and context

Closes #3183 by adding `is_active` filter to user list.

### How has this been tested?

Tested with api swagger.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by ActiveChooN — COMMENTED** [review]
Thanks for your contribution! Could you also include these changes to the cvat-core here and make the value true by default?
https://github.com/openvinotoolkit/cvat/blob/d14c26a32f35c75a899a0142bf93c81ddf502f4e/cvat-core/src/api-implementation.js#L115-L139

**Comment by ActiveChooN** [comment]
@dvkruchinin, looks like review pipeline tests failed, can take a look, please?

**Comment by dvkruchinin** [comment]
@ActiveChooN 
> looks like review pipeline tests failed, can take a look, please?

Sure. I`ll take a look.

**Comment by dvkruchinin** [comment]
@ActiveChooN 
The user's search query has changed.
In the file ```tests/cypress/support/commands_review_pipeline.js```
need to replace the line number 160 to
```cy.intercept('GET', `/api/v1/users?is_active=true&search=${user}&limit=10`).as('searchUsers');```

**Comment by Jooong** [comment]
> @ActiveChooN
> The user's search query has changed.
> In the file `tests/cypress/support/commands_review_pipeline.js`
> need to replace the line number 160 to
> `` cy.intercept('GET', `/api/v1/users?is_active=true&search=${user}&limit=10`).as('searchUsers'); ``

Also pushed this fix.

**Review by ActiveChooN — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Support of context images for 2D image tasks (<https://github.com/openvinotoolkit/cvat/pull/3122>)
+- Added is_active filter for user list (<https://github.com/openvinotoolkit/cvat/pull/3235>)
```
Comment (nmanovic) [inline CHANGELOG.md:13]:
```suggestion
- Filter `is_active` for user list (<https://github.com/openvinotoolkit/cvat/pull/3235>)
```

**Review by bsekachev — APPROVED** [review]

**Comment by bsekachev** [comment]
Thanks for the contribution!

**Pull request merged**